kernel-util/src/main/java/org/sakaiproject/util/Validator.java

```diff
@@ -359,7 +359,27 @@ public static String limitFormattedText(String value, int length)
         return ret.toString();
     }
 
- 
+	/**
+	Checks whether the email address has got valid local domain
+	@param email
+	 */
+	public static boolean isValidDomain(String email) {
```

Comment:
Although the configuration value is called invalid, that to me normally means syntactically valid, whereas this is specifically checking it's it's allowed so maybe `isAllowedEmailDomain(String email)`? Dunno.